samples/logging/prometheus-grafana-opentelemetry/Core_8/Endpoint/Program.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Threading.Tasks;
+using NServiceBus;
+using OpenTelemetry;
+using OpenTelemetry.Metrics;
+
+public class Program
+{
+    public static async Task Main()
+    {
+        var meterProviderBuilder = Sdk.CreateMeterProviderBuilder();
+
+        #region enable-opentelemetry-metrics
```

Comment:
should this include line 11 to be more clear about how to call this API in the first place?

Reply:
Updated